pkg/acsengine/defaults.go

```diff
@@ -247,6 +247,12 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 				a.OrchestratorProfile.KubernetesConfig.CloudProviderRateLimitBucket = DefaultKubernetesCloudProviderRateLimitBucket
 			}
 		}
+
+		// default etcd version
```

Comment:
As discussed on Slack, this is not called from some unit tests, and this may be what causes `make test` to fail.  This is a flaw in the tests but would be hard to fix.  As a workaround, it would be good to try adding `"etcdVersion": "2.5.2"` to the `const ExampleAPIModel` JSON at `deploy_test.go` line 18.  This would be quick to try, and if it works it would avoid having failing tests in master which everyone would need to know "oh yeah you can ignore that."

Reply:
Thanks for this, I added it. Now tests are went back to normal. On a different note, there has to be away to pickup pkg/acsengine/testdata